plotIt_ZA/listHistos.py

```diff
@@ -298,91 +299,40 @@ def get_flavour(name):
         elif "cosThetaStar" in key_name:
             plot['x-axis'] = "cos(#theta^{*}_{CS})_{lljj#slash{E}_{T}}"
             plot.update(defaultStyle_events)
-        
-        elif "DY_BDT" in key_name:
-            plot['x-axis'] = "DY reweighting BDT"
-            plot['legend-position'] = [0.2, 0.52, 0.53, 0.80]
-            plot.update(defaultStyle_events)
-        
+ 
         elif "MT2" in key_name:
             plot['x-axis'] = "MT2"
             plot.update(defaultStyle_events)
             if should_be_blind(key_name):
                 plot['blinded-range'] = [150, 500]
 
         elif "lljj_M_" in key_name:
-            plot['x-axis'] = "m_{lljj}"
+            plot['x-axis'] = "m_{lljj} (GeV)"
             plot.update(defaultStyle_events_per_gev)
             if should_be_blind(key_name):
                 plot['blinded-range'] = [500, 1500]
 
         elif "ll_M_" in key_name:
             plot['x-axis'] = "m_{ll} (GeV)"
             plot.update(defaultStyle_events_per_gev)
-            
+ 
             #### Do the yields here
             btag_stage = ""
             if "btagM" in key_name:
                 btag_stage = "llbb"
             else:
                 btag_stage = "lljj"
-            stage = ""
-            if "inverted_mll_cut" in key_name:
-                stage = ", inverted mll cut"
-            elif "mll_cut" in key_name:
-                stage = ", mll cut"
-                plot['for-yields'] = True
-            elif "mll_peak" in key_name:
-                stage = ", mll peak"
-            plot['yields-title'] = get_flavour(key_name) + ", " + btag_stage + stage
+            plot['yields-title'] = get_flavour(key_name) + ", " + btag_stage
+            plot['for-yields'] = True
             if args.yields:
                 plots['override'] = True
-        
-        elif "jj_M_" in key_name and "_vs_" not in key_name:
+
+        elif "jj_M_" in key_name and "_vs_" not in key_name and ("jj_deepCSV" not in key_name or "jj_cmva" not in key_name):
```

Comment:
shouldn't it be `"jj_deepCSV" not in key_name AND "jj_cmva" not in key_name"` ?

Reply:
No no, it's correct like that :)

Reply:
ok, understood